sched/scheduler/scheduler_test.go

```diff
@@ -2,6 +2,8 @@ package scheduler
 
 import (
 	"fmt"
+	"testing"
```

Comment:
I don't really look at this beyond the block syntax. I let goimports and gofmt handle this. I'm not sure those tools even let you decide how to arrange it?